packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentMethods.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { shape, string, bool } from 'prop-types';
+import { RadioGroup } from 'informed';
+import { usePaymentMethods } from '@magento/peregrine/lib/talons/CheckoutPage/PaymentInformation/usePaymentMethods';
+
+import Radio from '../../RadioGroup/radio';
+import CreditCardPaymentMethod from './creditCardPaymentMethod';
+import PaypalPaymentMethod from './paypalPaymentMethod';
+import { mergeClasses } from '../../../classify';
+
+import paymentMethodsOperations from './paymentMethods.gql';
+
+import defaultClasses from './paymentMethods.css';
+
+const PaymentMethods = props => {
```

Comment:
I don't think we should be using this query. It might be a false positive. Depending on the store it might return payment methods that are not handled in the current checkout workflow. We only support Credit Card as of now. We can and should add this query in the future and render the components based on the return value. Also `braintree` in the case above can only be a braintree credit card component, not every payment method that it supports.

I will need more info from @soumya-ashok and @jimbo on this before proceeding.

Reply:
I don't think pinging folks in Github is working. Can you set up some time with them to discuss this?

Reply:
@revanth0212 It is hard to keep track of github mentions and I had missed this one. Per our discussion with @awilcoxa I believe the only payment method we are supporting first is credit card, and possibly including others as a follow up. I am not sure if there would be a change to that approach based on what payment methods braintree supports. In any case, since design effort would be required to construct a good experience around multiple payment methods, it would likely need to be a follow up anyway.

Reply:
I'm fine with hardcoding one method (Braintree credit card) for now, since that's what we've built.